### PR TITLE
Adding Juice to the final main program

### PR DESCRIPTION
**Pull request by santhoshrajr**
## Summary

Adding missing juice to the final main program

Fixes dotnet/docs#15281

### CODE REVIEW

File: snippets/csharp/tour-of-async/AsyncBreakfast-final/Program.cs [hunk 1]
```diff
@@ -35,6 +35,8 @@ static async Task Main(string[] args)
                 }
                 allTasks.Remove(finished);
             }
+            Juice oj = PourOJ();
+            Console.WriteLine("oj is ready");
```
Comment (pkulikov) [inline snippets/csharp/tour-of-async/AsyncBreakfast-final/Program.cs]:
```suggestion
            Console.WriteLine("Orange juice is ready");
```

Comment (santhoshrajr) [reply snippets/csharp/tour-of-async/AsyncBreakfast-final/Program.cs]: The docs page has oj is ready in several other places. Should we change others as well or just this one?

Comment (pkulikov) [reply snippets/csharp/tour-of-async/AsyncBreakfast-final/Program.cs]: I didn't know about other occurrences. With them, I think the consistency should be maintained. The simplest way to do that is to ignore my suggestion and leave `oj` as it is.

Comment (santhoshrajr) [reply snippets/csharp/tour-of-async/AsyncBreakfast-final/Program.cs]: Thanks. Resoling the comment

**Comment by santhoshrajr** [comment]
Thank you @BillWagner

**Pull request merged**